persistence-jdbc/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/jdbc/JdbcPersistenceSpec.scala

```diff
@@ -29,16 +29,16 @@ abstract class JdbcPersistenceSpec(_system: ActorSystem) extends ActorSystemSpec
   import system.dispatcher
 
   protected lazy val slick = new SlickProvider(system)
-  protected lazy val session: JdbcSession = new JdbcSessionImpl(slick)
-  protected lazy val offsetStore = new JavadslJdbcOffsetStore(
-    slick,
-    system,
-    new OffsetTableConfiguration(
-      system.settings.config,
+  protected lazy val offsetStore =
```

Comment:
I move it out of the constructor to align with the code on the Slick side. It's easier to compare if they have the same structure

Reply:
now that I see it here, I think it's better to do it the other way around, declare this in one block to avoid multiplication of `lazy val`